pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -91,6 +91,9 @@
  *     <ul>
  *       <li>"/segments/{tableName}/{segmentName}/reload": reload a segment</li>
  *       <li>"/segments/{tableName}/reload": reload all segments</li>
+ *       <li>"/segments/{tableNameWithType}/{segmentName}/reset": reset a segment</li>
```

Comment:
May be expand on what is reset?

Reply:
This is just an overview of all the APIs. The detailed explanation is in the method javadoc.